devlab/tests/generate_load.py

```diff
@@ -642,29 +650,39 @@ def wait_until_vms_all_deleted():
             except Exception as e:
                 print "Network %s failed to delete: %s" % (network['name'],
                                                            repr(e))
-        for snapshot in config.snapshots:
+        for snapshot in self.config.snapshots:
             try:
                 self.glanceclient.images.delete(
                     self.get_image_id(snapshot['image_name']))
             except Exception as e:
                 print "Image %s failed to delete: %s" % (
                     snapshot['image_name'], repr(e))
-        for user in config.users:
+
+        sgs = self.neutronclient.list_security_groups()['security_groups']
+        for sg in sgs:
+            try:
+                print "delete sg {}".format(sg['name'])
+                self.neutronclient.delete_security_group(self.get_sg_id(
+                                                         sg['name']))
+            except Exception as e:
```

Comment:
Same as above. Please consider narrowing down the list of exceptions you want to handle here (maybe `neutronclient.exceptions.BaseException`?..)